Tutorials/CNTK_209A_Evaluation_of_Pretrained_Super-resolution_Models.ipynb

```diff
@@ -0,0 +1,634 @@
+    "                \n",
+    "        outfile = os.path.join(save_folder, entry)\n",
+    "            \n",
+    "        print(\"Now creating: \" + outfile)\n",
+    "            \n",
+    "        #function calls for different models\n",
+    "        if(i < 2):\n",
+    "            #residual learning, image is pre-upscaled and then cleared up\n",
+    "            evaluate(filename, models[i], outfile, output_dims[i], pre_upscale = True, clear_up = True, residual_model = True)\n",
+    "        else:\n",
+    "            #all upscaling is within the model\n",
+    "            evaluate(filename, models[i], outfile, output_dims[i], pre_upscale = False, clear_up = False, residual_model = False)\n",
+    "        \n",
+    "    #loop through models which can additionally clear up image after we increased it (DRNN and VDSR)\n",
+    "    for j in range(2):\n",
+    "        #loop through results of previously applied model\n",
+    "        for entry in os.listdir(save_folder):\n",
+    "            filename = os.path.join(save_folder, entry)\n",
+    "            filter_folder = os.path.join(results_folder, model_names[j] + \"_\" + model_names[i] + \"_results\")\n",
+    "                \n",
+    "            if not os.path.exists(filter_folder):\n",
+    "                os.makedirs(filter_folder)\n",
+    "                \n",
+    "            outfile = os.path.join(filter_folder, entry)\n",
+    "                \n",
+    "            print(\"Now creating: \" + outfile)\n",
+    "                \n",
+    "            #additionally clear up image without pre-magnifying\n",
+    "            evaluate(filename, models[j], outfile, output_dims[j], pre_upscale = False, clear_up = True, residual_model = True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false,
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "## Result analysis and future work\n",
+    "<p>Below is the example of what we were able to get. There is no mathematical formula which would evaluate how good our models perform. The best and most used method would be through opinion scoring, that is, presenting generated images to people so they can grade them from 1 to 5 depending on how realistic they look.</p>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {
+    "collapsed": false,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<img src=\"https://superresolution.blob.core.windows.net/superresolutionresources/example.PNG\"/>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from IPython.display import Image\n",
+    "Image(url = \"https://superresolution.blob.core.windows.net/superresolutionresources/example.PNG\")"
```

Comment:
We are currently showing images from our blob storage. How do we get images to cntk storage?